src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -1757,4 +1807,14 @@ public boolean mustWhenWillClaimBePaidBeShown(CaseData caseData) {
 
         return false;
     }
+
+    private String defendantFlagSpec(CaseData caseData, UserInfo userInfo) {
```

Comment:
The return value of this method is unused, which would be the whole point.

There's definitely something wrong here.

Also, see if this can be extracted to a base class, like the one in RespondToClaimCallbackHandler.

Reply:
same as RespondToClaimCallbackHandler